schemas/context/opportunitycomposite.example.1.json

```diff
@@ -0,0 +1,36 @@
+{
+  "xdm:opportunityID": "12345",
+  "xdm:opportunityType": "business",
+  "xdm:accountID": "89765",
+  "xdm:campaignID": "95678",
+  "externalSourceSystemAuditDetails":{
+  "xdm:createdBy": "USER111",
+  "xdm:createdDate": "2018-01-02T15:52:25+00:00",
```

Comment:
date example should explicitly be called out with dd > 12. Modify all the examples.